### PR TITLE
chore: removed extra wrappers for home page and rename health endpoint

### PR DESCRIPTION
**Pull request by IlyaBondar**
Issue #179 : Create api/health endpoint

### CODE REVIEW

**Review by denys-kolomiitsev — DISMISSED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Deployment status: success

**Review by akryvasheyeu — DISMISSED** [review]

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1 +1,229 @@
...
+              {enabledFeatures.includes('prompts-section') && <Promptbar />}
+              {isProfileOpen && <UserMobile />}
+            </div>
+          </div>
+        </main>
+      )}
+    </>
+  );
+};
+export default Home;
```
Comment (mikitabut) [inline src/pages/index.tsx]: Do you need this anymore?

Comment (IlyaBondar) [reply src/pages/index.tsx]: @mikitabut, yes, it's just name of our default page which can be any and used only inside this file.
We export page(`default`, name doesn't matter) and `getServerSideProps`

**Review by mikitabut — APPROVED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Deployment status: success

**Comment by alexey-ban** [comment]
/deploy-review
>Deployment status: failed

**Pull request merged**